src/main/java/com/fabriccommunity/thehallow/mixin/client/ClientPlayNetworkHandlerMixin.java

```diff
@@ -25,4 +31,28 @@ private void redirectOpenScreen_onPlayerRespawn(MinecraftClient client, Screen s
 			client.openScreen(screen);
 		}
 	}
+
+	@Shadow
+	private ClientWorld world;
+
+	@Inject(at = @At("TAIL"), method = "onEntitySpawn")
+	public void assignProxy(EntitySpawnS2CPacket packet, CallbackInfo info) {
+		if (packet.getEntityTypeId() == HallowedEntities.SHOTGUN_PROJECTILE) {
```

Comment:
I think we use fabric packet registry in this mod instead of arbitrary inject to vanilla spawn packets